test/main/services/runtime/ClaudeMultimodelBridgeService.test.ts

```
@@ -2114,6 +2115,10 @@ describe('ClaudeMultimodelBridgeService', () => {
   });
 
   it('keeps global and project-scoped OpenCode status passive and independent', async () => {
+    const scopedProjectPath = '/tmp/scoped-project';
```

ESLint

[error] Make sure publicly writable directories are used safely here.

(sonarjs/publicly-writable-directories)